# 

Header header
int32 frame_w
int32 frame_h
int32[] cx
int32[] cy
string[] type
string[] text